File: Modul 05/04/PRAK504-2410817310020-M. Rizki Dinar.py
import math
from decimal import Decimal, getcontext
getcontext().prec = 1

def reverse(val):
    subs = 1
    clean = 0
    multiplier = 1
    for i in range(len(str(val))):
        multiplier *= 10
    
    while subs >= 1:
        subs = math.floor(val/10)
        dirt = float(Decimal(str(val/10)) - Decimal(subs)) * multiplier
        clean += math.floor(dirt)
        multiplier /= 10
        val = subs
    return clean

A,B = map(int, input().split())
A = reverse(A)
B = reverse(B)
C = A+B
print(reverse(C))

File: Modul 05/04/PRAK504-2410817310020-M. Rizki Dinar.c
#include <stdio.h>
#include <math.h>
#include <stdlib.h>
int reverse(int val){
    double dirt;
    int clean = 0;
    int subs = 1;
    int multiplier = 1;
    
    int digit = floor(log10(abs(val)))+1;
    for(int i = 0; i < digit; i++){
        multiplier *= 10;
    }
    
    while(subs >= 1){
        subs = floor(val/10);
        dirt = ((double)val/10 - subs) * multiplier;
        clean += round(dirt);
        multiplier /= 10;
        val = subs;
    }
    return clean;
}
int main() {
    int A, B;
    scanf("%d %d",&A,&B);
    A=reverse(A);
    B=reverse(B);
    int C = A+B;
    printf("%d",reverse(C));
    return 0;
}